model/event/DeliveryCreatedEvent.php

```diff
@@ -44,7 +44,7 @@ class DeliveryCreatedEvent extends AbstractDeliveryEvent implements WebhookSeria
      *
      * @throws \core_kernel_persistence_Exception
      */
-    public function __construct(core_kernel_classes_Resource $delivery, core_kernel_classes_Resource $originTest)
+    public function __construct(core_kernel_classes_Resource $delivery, ?core_kernel_classes_Resource $originTest =  null)
```

Comment:
Perhaps, it's a good time to update or remove the outdated docblock of this method.

Reply:
Done